great-idea-website/css/index.css

```diff
@@ -63,4 +63,153 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+body {
+	width:900px;
+	margin: auto;
+}
+nav {
+	width: 100%;
+	margin-top: 10px;
+	position: relative;
+	display: inline-block;
+}
+
+nav .nav-items {
+	width: 65%;
+	display: inline-block;
+	position: relative;
+	text-align: center;
+	vertical-align: middle;
+	margin-bottom: 16px;
+}
+
+nav .nav-items a {
+	display: inline-block;
+	position: relative;
+	padding-left: 10px;
+	padding-right: 30px;
+	font-size: 16px;
+	text-decoration: none;
+	color: grey;
+}
+
+nav .logo {
+	width: 25%;
+	position: relative;
+	margin: 25px 40px 30px 40px;
+	padding: 5px 5px 5px 5px;
+	vertical-align: middle;
+	;
+}
+
+header {
+	width: 100%;
+	position: relative;
+	display: inline-block;
+	margin-bottom: 20px;
+	border-bottom: 1px solid black; 
+}
+
+header  .getting-started {
+	width: 49.5%;
+	position: relative;
+	display: inline-block;
+	margin: 50px auto;
+	padding: 50px;
+	text-align: center;
+	vertical-align: middle;
+	font-size: 90px;
+}
+
+header .getting-started .start {
+	position: relative;
+	display: inline-block;
+	font-size: 16px;
+	padding: 10px 20px;
+	border: 1px solid black;
+	text-align: center;
+}
+
+header .snippet {
+	width: 49.5%;
+	position: relative;
+	display: inline-block;
+	margin: 20px auto;
+	text-align: center;
+	vertical-align: middle;
+}
+
```

Comment:
I like your class naming convention, but there is room for improvement. top middle and bottom are descriptive enough that I can tell what you are pointing at by looking at your design file and the page. You do repeat yourself a bit with the divs within those larger boxes. Notice how you apply the same styling to features and about. One thing you should strive for in your coding is to keep it D.R.Y. (Don't Repeat Yourself). So you could either do something like 'main .top .features, main .top .about {...}' or  you could give both of those divs the same class name. Same with the divs in the bottom section